instrumentation/kamon-akka-http/src/main/scala/kamon/instrumentation/akka/http/AkkaHttpClientInstrumentation.scala

```diff
@@ -33,7 +33,10 @@ class AkkaHttpClientInstrumentation extends InstrumentationBuilder {
     * Simply modifies the requests as they are submitted. This does not cover connection pooling, just requests sent
     * via the Http.singleRequest mechanism.
     */
+
   onType("akka.http.scaladsl.HttpExt")
+    .advise(method("singleRequest"), classOf[HttpExtSingleRequestAdvice])
+    // akka-http 10.1.x compatibility
```

Comment:
I'm not totally sure about this. If `singleRequest` ends up calling `singleRequestImpl` in 10.1.x, wouldn't this be causing the instrumentation create two spans for all client requests?